src/scribe_data/utils.py

```diff
@@ -86,27 +153,15 @@ def get_language_iso(language: str) -> str:
 
     Returns
     -------
-        The ISO code for the language as a value of a dictionary.
+        str
+            The ISO code for the language.
     """
-    language = language.lower()
-
-    language_iso_dict = {
-        "english": "en",
-        "french": "fr",
-        "german": "de",
-        "italian": "it",
-        "portuguese": "pt",
-        "russian": "ru",
-        "spanish": "es",
-        "swedish": "sv",
-    }
-
-    if language not in language_iso_dict:
-        raise ValueError(
-            f"{language.capitalize()} is currently not a supported language for ISO conversion."
-        )
-
-    return language_iso_dict[language]
+    return _find(
```

Comment:
This is what I'm talking about in #55. We could basically get rid of this function and use langcodes, thus allowing us to add languages without needing to track their ISOs :)